Lib/asyncio/events.py

```diff
@@ -649,7 +632,7 @@ def get_event_loop(self):
         """
         if (self._local._loop is None and
                 not self._local._set_called and
-                threading.current_thread() is threading.main_thread()):
+                isinstance(threading.current_thread(), threading._MainThread)):
```

Comment:
I dind't check deeply but, the intention of the original code is check fi current_thread() is the same object of main_thread(). Why are you change it?